Xamarin.UserConsent/UI/ConsentFormPage.xaml

```diff
@@ -19,34 +19,36 @@
     <Frame StyleClass="consentPopup"
            VerticalOptions="Center"
            HorizontalOptions="Center">
-        <StackLayout VerticalOptions="Center" 
+        <ScrollView>
```

Comment:
FYI - this is the important addition

Reply:
@mgawron-fp you need to bump the version so that it gets published.

Reply:
done